.github/workflows/testAndPublish.yml

```diff
@@ -58,13 +60,16 @@ jobs:
       run: ci/scripts/setBuildVersionVars.ps1
     - name: Set scons args
       run: ci/scripts/setSconsArgs.ps1
-    - name: set cache key for SCons MSVC Cache
+    - name: Set cache key for SCons MSVC Cache
       shell: bash
       run: echo "SCONS_CACHE_KEY=${RUNNER_OS}-${ImageVersion}" >> $GITHUB_ENV
+    # On GithubActions, it can take more than two minutes for SCons to collect information about MSVC tooling, majorly impacting build performance.
```

Comment:
```suggestion
    # On Github Actions, it can take more than two minutes for SCons to collect information about MSVC tooling, majorly impacting build performance.
```